Purpose: O_DIRECT is undefined on Debian/hurd.
Fixes:   #534471
Authour: Samuel Thibault <sthibault@debian.org>
Status:  Submitted upstream.
---
--- a/makeboot.c
+++ b/makeboot.c
@@ -20,6 +20,9 @@
 #ifndef POSIX_FADV_SEQUENTIAL
 #define posix_fadvise(fd, off, len, adv)	(-1)
 #endif
+#ifndef O_DIRECT
+#define O_DIRECT 0
+#endif
 static int o_flags = O_RDONLY;
 #endif
 
